internal/config/config.go

```diff
@@ -37,21 +38,23 @@ func parseClientFlags(c *ClientConfig) {
 	flag.Parse()
 }
 
-func (s *ServerConfig) New() ServerConfig {
+func (s *ServerConfig) New() *ServerConfig {
```

Comment:
```suggestion
func  NewServer() *ServerConfig {
```
Обсудили что конструктор должен быть функцией а не методом

Reply:
Реализовал инициализацию клиента и сервера через отдельные функции